eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -302,7 +302,7 @@ jobs:
 
     - pwsh: >-
         Copy-Item
-        sdk/docs/core/CHANGELOG.md
+        CHANGELOG.md
```

Comment:
@danieljurek - I am not familiar with the infra/tool side of things, so please let me know if this change needs to be made elsewhere (in azure tools/etc.), or if it should be made in this repo, as is.